jbpm-installer/src/main/resources/build.properties

```diff
@@ -27,11 +27,11 @@ jBPM.url=https://origin-repository.jboss.org/nexus/service/local/artifact/maven/
 # for EAP7:
 #   jBPM.console.url=https://origin-repository.jboss.org/nexus/content/groups/public-jboss/org/kie/business-central/${jBPM.version}/business-central-${jBPM.version}-eap7.war
 # for WildFly19:
-#   jBPM.console.url=https://origin-repository.jboss.org/nexus/content/groups/public-jboss/org/kie/business-central/${jBPM.version}/business-central-${jBPM.version}-wildfly19.war
+#   jBPM.console.url=https://origin-repository.jboss.org/nexus/content/groups/public-jboss/org/kie/business-central/${jBPM.version}/business-central-${jBPM.version}-wildfly23.war
 # or:
 #<SNAPSHOT>
-#   jBPM.console.url=https://origin-repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=business-central&v=${jBPM.version}&e=war&c=wildfly19
-jBPM.console.url=https://origin-repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=business-central&v=${jBPM.version}&e=war&c=wildfly19
+#   jBPM.console.url=https://origin-repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=business-central&v=${jBPM.version}&e=war&c=wildfly23
+jBPM.console.url=https://origin-repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=business-central&v=${jBPM.version}&e=war&c=wildfly23
```

Comment:
we should replace origin repository whenever it is still referenced I think

Reply:
DONE